constants.ini

```diff
@@ -19,4 +19,4 @@ OUTLOOK_CLIENT_ID=value
 OUTLOOK_SECRET_KEY=value
 OUTLOOK_REFRESH_KEY=value
 OUTLOOK_SCOPE=User.Read Mail.ReadWrite MailboxSettings.ReadWrite offline_access
-DB_PASSWORD=value
\ No newline at end of file
+DB_PASSWORD=ad@1fgbR!sd
```

Comment:
Any reason we have changed it from being "value"?

Reply:
no reason it should have valid db password when runing scripts,since we discuss this in meeting i added a random value